### DISCUSSION TITLE
Custom Pages and Resources

### DISCUSSION THREAD

**Discussion by Peagah-Vieira** [buildix/timex#28] [Q&A]
I'm having problems trying to customize the pages and resources, even changing the settings it keeps the original

**Comment by mikr0smile** [comment]
Hello. Are you talking about filament Resources? Yes, I found an issue, and I'll fix it. 

Regarding Calendar itself (Timex page) - you can create custom filament page and extends Buildix\Timex\Pages\Timex, and register your custom page in timex config, so there won't be duplicates in navigation.

**Reply by haringsrob** [reply]
This unfortunately does not work I have noticed.

In Day.php there is a static call to Timex::getEvents() so the custom page override of that method is being ignored.

**Reply by mikrosmile** [reply]
> This unfortunately does not work I have noticed.
> 
> 
> 
> In Day.php there is a static call to Timex::getEvents() so the custom page override of that method is being ignored.

Okay, thank you. 
I'll consider another approach.

**Reply by Peagah-Vieira** [reply]
Hello, for **custom pages** we can extends Buildix\Timex\Pages\Timex like @mikrosmile said, but for **custom resources**, I needed to build a resource normally and just import timextraits.

**Custom Page**
![buildix_pages](https://user-images.githubusercontent.com/105545343/217342561-2977719a-4bdb-4ce2-baa2-1c53fb4a47a7.png)

**Custom Resource**
![buildix_events](https://user-images.githubusercontent.com/105545343/217342576-4b978f02-3a09-4b20-8825-8f51f763c138.png)
